test/src/unit-sparse-global-order-reader.cc

```diff
@@ -153,67 +505,46 @@ void CSparseGlobalOrderFx::update_config() {
           &error) == TILEDB_OK);
   REQUIRE(error == nullptr);
 
-  REQUIRE(
-      tiledb_config_set(
-          config, "sm.mem.total_budget", total_budget_.c_str(), &error) ==
-      TILEDB_OK);
-  REQUIRE(error == nullptr);
-
-  REQUIRE(
-      tiledb_config_set(
-          config,
-          "sm.mem.reader.sparse_global_order.ratio_tile_ranges",
-          ratio_tile_ranges_.c_str(),
-          &error) == TILEDB_OK);
-  REQUIRE(error == nullptr);
-
-  REQUIRE(
-      tiledb_config_set(
-          config,
-          "sm.mem.reader.sparse_global_order.ratio_array_data",
-          ratio_array_data_.c_str(),
-          &error) == TILEDB_OK);
-  REQUIRE(error == nullptr);
-
-  REQUIRE(
-      tiledb_config_set(
-          config,
-          "sm.mem.reader.sparse_global_order.ratio_coords",
-          ratio_coords_.c_str(),
-          &error) == TILEDB_OK);
-  REQUIRE(error == nullptr);
+  REQUIRE(memory_.apply(config) == nullptr);
 
   REQUIRE(tiledb_ctx_alloc(config, &ctx_) == TILEDB_OK);
```

Comment:
Since you added a vfs_test_setup_ I'd expect not to allocate a fresh context but update the vfs_test_setup_.ctx_ instead and set ctx_ to that. Something like:

Reply:
Thanks for the link!

9ea370e7